charts/nri-bundle/requirements.yaml

```
@@ -1,13 +1,21 @@
 dependencies:
+  # Old newrelic-infrastructure chart, now deprecated.
   - name: newrelic-infrastructure
     repository: file://../newrelic-infrastructure
     condition: infrastructure.enabled
     version: 2.7.3
 
+  # New newrelic-infrastructure chart.
+  - name: newrelic-infrastructure-v3
+    alias: newrelic-infrastructure
+    repository: file://../newrelic-infrastructure-v3
+    condition: newrelic-infrastructure-v3.enabled
```

I would change this.

Now we have something like
```yaml
infrastructure:
  enabled: true
newrelic-infrastructure:
  integrations_config:
    - name: nri-redis.yaml
      data:
        [...]
```
And we are changing it to
```yaml
newrelic-infrastructure-v3:
  enabled: true
newrelic-infrastructure:
  integrations_config:
    - name: nri-redis.yaml
      data:
        [...]
```

with my proposed change we can change it now to something more idiomatic like
```yaml
newrelic-infrastructure:
  enabled: true
  integrations_config:
    - name: nri-redis.yaml
      data:
        [...]
```
```suggestion
    condition: newrelic-infrastructure.enabled
```